vcd/resource_vcd_vapp_nat_rules_test.go

```diff
@@ -64,17 +64,22 @@ func TestAccVcdVappNatRules(t *testing.T) {
 			resource.TestStep{
 				Config: configText,
 				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "enabled", "true"),
```

Comment:
Just a thought. How do you make sure that FW is enabled during this test? As without the FW enabled, NAT enable doesn't work.

Reply:
I was wrong, API allows to enable, but in Env it may not work(even it shows enabled) as UI shows warning.

Reply:
I concur, after more experimenting I observed the same behavior. You can enable NAT with disabled FW, but then in UI "Edit" you see warnings:
> The NAT service is unavailable because the firewall is disabled. To enable the NAT service, you must enable the firewall.

Reply:
From test perspective nothing change even I explicitly enable FW

Reply:
added